sdk/cosmos/azure-cosmos/azure/cosmos/aio/cosmos_client.py

```diff
@@ -166,8 +180,7 @@ def from_connection_string(cls, conn_str, credential=None, consistency_level=Non
         )
 
     @staticmethod
-    def _get_database_link(database_or_id):
-        # type: (Union[DatabaseProxy, str, Dict[str, str]]) -> str
+    def _get_database_link(database_or_id: Union[DatabaseProxy, str, Dict[str, str]]) -> str:
```

Comment:
Sometimes you are using `Dict[str, str]` and others `Dict[str, Any]`.  Is this intentional?

Reply:
yep, depends on what's being returned; in this case we know for sure it's a string to string mapping, while for most of the others the body could also contain str to int or bool mappings.